tests/core/test_migrator.py

```diff
@@ -0,0 +1,220 @@
+import uuid
```

Comment:
the coverage report also suggests to test the return value of `publish_events()`

Reply:
ah, apparently tested by the `test_export` module 👍